kubos/versions.py

```diff
@@ -49,9 +49,10 @@ def get_tag_list(repo):
 def print_tag_list(tag_list):
     active_version = get_active_kubos_version()
     for tag in tag_list:
+        tag_name = tag.name #tag.name is immutable...
         if tag.name == active_version:
-            sys.stdout.write('*')
-        print tag.name
+            tag_name = tag.name + ' *'
+        logging.info(tag_name)
 
 
 def get_latest_tag(tag_list):
```

Comment:
This `get_latest_tag` function (and maybe these other tag functions?) look like good candidates to move to the git util module.